src/client/firefox.js

```diff
@@ -8,11 +8,19 @@ import { setupCommands, clientCommands } from "./firefox/commands";
 import { setupEvents, clientEvents } from "./firefox/events";
 import { features } from "../utils/prefs";
 
+let DebuggerClient;
+
+function createObjectClient(grip: Object) {
```

Comment:
could we use the Grip type here from flow-typed/debugger-html? https://github.com/devtools-html/debugger.html/blob/master/flow-typed/debugger-html.js#L235

Reply:
sure